Add more applications (#113)
jump-dev/jump-dev.github.io: index.md

## JuMP is used...

 * to model and solve real-world problems, including

>     * large-scale inventory routing problems at [Renault](https://arxiv.org/abs/2209.00412)

Author: ```suggestion
    * large-scale inventory routing problems at [Renault](https://pubsonline.informs.org/doi/10.1287/trsc.2022.0342)
```

    * power grid expansion planning at [PSR](https://juliacomputing.com/case-studies/psr/)
    * school bus routing for [Boston Public Schools](https://www.the74million.org/article/building-a-smarter-and-cheaper-school-bus-system-how-a-boston-mit-partnership-led-to-new-routes-that-are-20-more-efficient-use-400-fewer-buses-save-5-million/)
 * to teach optimization at universities around the world, including
    * [MIT](https://orc.mit.edu)
    * [DTU](https://www.man.dtu.dk/mathprogrammingwithjulia)
    * [École des Ponts](https://ecoledesponts.fr)
    * [KAIST](https://kaist.ac.kr/en/)
    * [U. Wisconsin-Madison](https://engineering.wisc.edu/departments/industrial-systems-engineering/)
    * [PUC-Rio](http://www.puc-rio.br/english/)
    * [Université de Nantes](https://www.univ-nantes.fr)
 * to build continental-scale energy system models, including
     * MIT's [GenX](https://github.com/GenXProject/GenX)
     * the National Renewable Energy Laboratory's [Sienna](https://www.nrel.gov/analysis/sienna.html)
     * Spine's [SpineOpt.jl](https://www.tools-for-energy-system-modelling.org)
     * TNO's [TulipaEnergyModel.jl](https://github.com/TulipaEnergy/TulipaEnergyModel.jl)

>  * to power exa-scale research into personalized medicine at

Author: ```suggestion
 * to accelerate pre-exascale metabolic modeling workflows at
```

## JuMP is used...

>  * to power exa-scale research into personalized medicine as part of [PerMedCoE](https://permedcoe.eu/core-applications/)

Author: cc @exaexa is this correct?

Reviewer: ```suggestion
 * to accelerate pre-exascale metabolic modeling workflows in
   [PerMedCoE project](https://permedcoe.eu/core-applications/)
```

thanks!

Author: What does `pre-exascale` mean?
